plugins/modules/system/lvol.py

```diff
@@ -325,7 +341,16 @@ def main():
         test_opt = ''
 
     if size:
-        # LVCREATE(8) -l --extents option with percentage
+        # LVEXTEND(8)/LVREDUCE(8) -l, -L options: Check for relative value for resizing
+        if '+' in size:
+            size_operator = '+'
+            size = size[1:]
+        elif '-' in size:
+            size_operator = '-'
+            size = size[1:]  
```

Comment:
This line could (should?) go outside of the `if`, it is the same for both cases.

Reply:
Since it doesn't show up in `else` as well, that won't work.

There's a problem with this `if`/`elif` though: it tests for `+`/`-` in the string, but then assumes it appeared as the first character. It should be `if size.startswith('+'):` instead of `if '+' in size:`.

Reply:
oh yeah, I was a happy trigger there... :facepalm: